pkg/binlog-filter/filter.go

```diff
@@ -35,11 +35,19 @@ type EventType string
 
 // show DML/DDL Events
 const (
+	DDL EventType = "ddl"
+	DML EventType = "dml"
```

Comment:
should we need to differ `DDL`, `DML` from other `EventType`s? because them only be used internal, and user should not config them